src/resources/js/aggregator-fields.js

```diff
@@ -1096,13 +1095,8 @@ tribe_aggregator.fields = {
 		}
 
 		if ( data.continue ) {
-			// If multiple editors are open for the same event we don't want to hammer the server
-			// and so a min delay of 1/2 sec is introduced between update requests
-			if ( elapsed < 500 ) {
-				setTimeout( obj.progress.send_request, 500 - elapsed  );
-			} else {
-				obj.progress.send_request();
-			}
+			setTimeout( obj.progress.send_request, obj.progress.increment  );
+			obj.progress.increment += 50;
```

Comment:
Drop a comment above these lines to explain what is happening here and the incrementing timeout mechanics and its reasons.

Reply:
No update on this one?

Reply:
Looks like Github didn't catch the update as it was on the line above.

https://github.com/moderntribe/the-events-calendar/blob/batch-pushing/src/resources/js/aggregator-fields.js#L1099-L1113